.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Standard Angular App Flow
```

Comment:
Try to follow the folder structure as used for osmox(we have apps folder).

Reply:
fixed it, foloowed osmox structure